Library/Formula/ploticus.rb

```diff
@@ -0,0 +1,77 @@
+class Ploticus < Formula
+  desc "Scriptable plotting and graphing utility"
+  homepage "http://ploticus.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/ploticus/ploticus/2.42/ploticus242_src.tar.gz"
+  version "2.42"
+  sha256 "3f29e4b9f405203a93efec900e5816d9e1b4381821881e241c08cab7dd66e0b0"
+
+  depends_on "libpng"
+
+  # Adds support for PREFIX in Makefile
+  patch :DATA
```

Comment:
Just realized the patch isn't actually needed; we can override `INSTALLBIN` directly, since that's the only thing under PREFIX it actually installs files to. Revised the PR to remove it. What do you think now?